example.config.yml

```diff
@@ -69,6 +69,19 @@ apache_vhosts:
   - {servername: "xhprof.drupalvm.dev", documentroot: "/usr/share/php/xhprof_html"}
   - {servername: "pimpmylog.drupalvm.dev", documentroot: "/usr/share/php/pimpmylog"}
 
+# Enable SSL Support on the Drupal site vhosts.
+# See https://github.com/geerlingguy/ansible-role-apache for additional options.
+# Update the file paths below if you are using a custom SSL certificate.
+apache_vhosts_ssl:
+  - {
+    servername: "{{ drupal_domain }}",
+#    serveralias: "www.{{ drupal_domain }}",
+    documentroot: "{{ drupal_core_path }}",
+    certificate_file: "/etc/ssl/certs/ssl-cert-snakeoil.pem",
```

Comment:
One difficulty here: this cert isn't available on CentOS (at least, not by default), and I'd like to have all examples OOTB work with either Ubuntu or CentOS (latest).

Maybe we can leave this example conf commented? Or at least rewrite comments a little to mention this fact (and add a note to the docs on different OSes).